draft-tschofenig-rats-psa-token.md

```diff
@@ -400,23 +415,71 @@ verifier may choose to ignore this claim in favor of other information.
 ~~~
 
 ### Profile Definition
+{: #sec-profile-definition-claim}
+
+The Profile Definition claim encodes the unique identifier that corresponds to
+the EAT profile described by this document.  This allows a receiver to assign
+the intended semantics to the rest of the claims found in the token.
+
+The EAT `profile` (claim key 18) is used.  The following constraints
+apply to its type:
 
-The Profile Definition claim contains the name of a document that describes the
-"profile" of the report. The document name may include versioning. The value
-for this specification MUST be PSA_IOT_PROFILE_1.
+* The URI encoding MUST be used;
+* The value MUST be `http://arm.com/psa/2`.
+
+This claim MUST be present in a PSA attestation token.
+
+See {{sec-backwards-compat}}, for considerations about backwards compatibility
+with previous versions of the PSA attestation token format.
 
 ~~~
 {::include cddl/psa-profile.cddl}
 ~~~
 
+# Backwards Compatibility Considerations
+{: #sec-backwards-compat}
+
+Previous versions of this specification used different keys for the following
+claims:
+
+* Nonce (claim key -75008);
+* Instance ID (claim key -75009);
+* Profile Description (claim key -75000 and value `PSA_IOT_PROFILE_1`).
+
+These claim keys have been retired and their use is deprecated.
```

Comment:
Maybe worth saying this choice was taken because IANA registration became available for standard claim values (rather than being an arbitrary decision we took for fun)!

Reply:
I had a sentence to that end, but I dropped it because I thought that it'd be a distraction.  We want cold, dull, uninspiring standards language :-)

Reply:
> I had a sentence to that end, but I dropped it because I thought that it'd be a distraction. We want cold, dull, uninspiring standards language :-)
This